gcp/copy_folder.py

```
@@ -234,17 +240,88 @@ def _dest_has_folder(svc, dest_folder_id, folder_name):
+    """
+    try:
+        results = (
+            svc.permissions()
+            .list(
+                fileId=src_id,
+                fields="permissions(id,role,type,emailAddress,domain,allowFileDiscovery)",
+            )
+            .execute()
+        )
+    except HttpError as err:
+        logging.error("Failed to list permissions for %s: %s", src_id, err)
```

Ruff (0.16.3)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)